tests/test_client.py

```diff
@@ -463,3 +463,61 @@ def additional_matcher(request):
                 hash=ORDER_HASH
             )
             assert result == json_obj
+
+    # ------------ replace_order ------------
+
+    def test_replace_order_success(self):
+
+        def additional_matcher(request):
+            body = json.loads(request.body)
+            assert body['fillOrKill'] is False
```

Comment:
should use `==` instead of `is` I believe

https://www.w3schools.com/python/ref_keyword_is.asp